backend/src/service/plantings.rs

```diff
@@ -1,14 +1,20 @@
 //! Service layer for plantings.
 
+use actix_http::StatusCode;
 use actix_web::web::Data;
+use chrono::Days;
 use uuid::Uuid;
 
 use crate::config::data::AppDataInner;
 use crate::error::ServiceError;
 use crate::model::dto::plantings::{
     NewPlantingDto, PlantingDto, PlantingSearchParameters, UpdatePlantingDto,
 };
-use crate::model::entity::plantings::Planting;
+use crate::model::dto::TimelinePage;
+use crate::model::entity::plantings::{FindPlantingsParameters, Planting};
+
+/// Time offset for loading plantings in the timeline.
```

Comment:
I think its supposed to be days?
```suggestion
/// Time offset in days for loading plantings in the timeline.
```